docs/contributing/index.md

```diff
@@ -1,17 +1,17 @@
 ---
-title: Contributing to Gatsby.js
-description: Learn about contributing to one of the most welcoming communities helping develop the future of the web
+title: Contribuer à Gatsby.js
+description: Apprenez en plus sur la contribution à l'une des communauté les plus acceuillante en aidant à developper le future d'internet.
 disableTableOfContents: true
 ---
 
-Thanks for being interested in contributing! We're so glad you want to help!
+Merci pour votre interet pour la contribution Nous somme heureux que vous souhaitiez nous aider !
```

Comment:
```suggestion
Merci de votre intérêt pour contribuer ! Nous sommes heureux que vous souhaitiez nous aider !
```

Reply:
I agree with everything except "pour contribuer". It sounds weird to me...